impeller/entity/shaders/blending/advanced_blend.glsl

```diff
@@ -24,12 +25,11 @@ in vec2 v_src_texture_coords;
 out f16vec4 frag_color;
 
 f16vec4 Sample(f16sampler2D texture_sampler, vec2 texture_coords) {
-// gles 2.0 is the only backend without native decal support.
-#ifdef IMPELLER_TARGET_OPENGLES
-  return IPSampleDecal(texture_sampler, texture_coords);
-#else
-  return texture(texture_sampler, texture_coords);
-#endif
+  if (blend_info.supports_decal_sampler_address_mode > 0.0) {
```

Comment:
These can still technically be ifdefs since vulkan/metal always support this.

Reply:
Got it, added them back.